.changeset/quiet-pears-float.md

```diff
@@ -0,0 +1,5 @@
+---
+'@apollo/server': patch
+---
+
+Update schemaIsSubgraph to also support non nullable \_Service.sdl
```

Comment:
@trevor-scheer @patrick91 Since this text shows up for end users in the CHANGELOG it would be best to change it (which can happen on main pre-release) to something that's a bit more user-focused. I would suggest that this is something more like "A few features of Apollo Server behave differently if the schema is a subgraph (for example, the inline trace plugin is installed by default in subgraphs). Previously, this logic did not properly detect subgraphs created by vXXX of `@apollo/server`; now, it does."

Also I think there's at least one place in the docs (https://www.apollographql.com/docs/apollo-server/api/plugin/inline-trace, maybe more) that says how this logic works that should be updated.

Reply:
https://github.com/apollographql/apollo-server/pull/7301